targets/PythonSdk/source/uploadPython.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# cd F:/sdks/PythonSDK/
+cd /mnt/f/sdks/PythonSDK/
+
+python setup.py sdist bdist_wheel
+
+if [ -p "$1" ]
+then
+    twine upload dist/*
+    exit 1
+fi
+    # last test version was 0.2.8 any earlier version uploaded will probably get stomped over by pypi's cache
```

Comment:
I think this can probably be un-indented.

Reply:
Indeed, tabbing in bash means nothing, and thus you can use it however you want to make your script more readable.

Reply:
Bash also has functions, which syntactically are basically C-like syntax.
Use them however makes sense to you.